### PR TITLE
BTFS-1219 add RemoteAPI setting

### PR DESCRIPTION
**Pull request by richlee33**
add the RemoteAPI setting to the iptb so it can automatically vary the port.

### CODE REVIEW

**Comment by richlee33** [comment]
iptb logs | grep Remote
```
Remote API server listening on /ip4/127.0.0.1/tcp/55576
Remote API server listening on /ip4/127.0.0.1/tcp/55572
Remote API server listening on /ip4/127.0.0.1/tcp/55591
Remote API server listening on /ip4/127.0.0.1/tcp/55588
Remote API server listening on /ip4/127.0.0.1/tcp/55592
```

**Review by taiyangc — COMMENTED** [review]
go fmt and we are good

File: localbtfs/localbtfs.go [hunk 1]
```diff
@@ -77,6 +84,14 @@ func NewNode(dir string, attrs map[string]string) (testbedi.Core, error) {
 		}
 	}
 
+        if remoteapiaddrstr, ok := attrs["remoteapiaddr"]; ok {
```
Comment (taiyangc) [inline localbtfs/localbtfs.go]: @richlee33 this file is missing fmt. Could you try from root folder `go fmt ./...` ?

Comment (richlee33) [reply localbtfs/localbtfs.go]: sorry, i'll do better next time.

**Review by taiyangc — APPROVED** [review]

**Pull request merged**